### PR TITLE
First version of the changelog

### PR DESCRIPTION
**Pull request by Fokko**

### CODE REVIEW

File: landing-page/content/common/release-notes.md [hunk 1]
```diff
@@ -70,8 +70,21 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
 
 ## 1.1.0 release
 
+Has been released on 28 November 2022. The 1.1.0 release deprecates various pre-1.0.0 methods:
+
+* Puffin statistics have been [added to the Table API](https://github.com/apache/iceberg/pull/4945)
+* Support for [Table scan reporting](https://github.com/apache/iceberg/pull/5268), which enables collection of statistics of the table scans.
+
```
Comment (ajantha-bhat) [inline landing-page/content/common/release-notes.md:77]: On top of my head these are worth mentioning:
1. Add dataSequenceNumber to ManifestEntry #5913
2. Support register table for all the catalogs (previously it was only for hive) #5037
3. Flink 1.16 support, flink 1.13 removal 
4. Spark-3.0 removal

Comment (Fokko) [reply landing-page/content/common/release-notes.md:77]: Great suggestions, added 👍🏻

File: landing-page/content/common/release-notes.md [hunk 2]
```diff
@@ -70,8 +70,25 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
 
 ## 1.1.0 release
 
+Has been released on 28 November 2022. The 1.1.0 release deprecates various pre-1.0.0 methods:
```
Comment (ajantha-bhat) [inline landing-page/content/common/release-notes.md:73]: I also remembered now that we can collect highlights from our iceberg sync document's highlights as @rdblue does a really good job of tracking them. 

Here are highlights that are missing from the current list. (I don't have tracking PR though, need to find it based on author's name)
Spark: Aggregate expressions were added (Thanks, Huaxin!)
Core: Added changelog table and new scan class (Thanks, Anton!)
Core: Added support for append/delete to branches (Thanks, Amogh!)
Core: Improved snapshot expiration (Thanks Szehon and Amogh!)
SnapshotProducer supports branch writes (Thanks, Namratha!)
FLIP-27 reader is supported in SQL (Thanks, Steven!)
Spark FunctionCatalog was added (Thanks, Kyle!)

File: landing-page/content/common/release-notes.md [hunk 3]
```diff
@@ -70,8 +70,25 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
...
+
+* Puffin statistics have been [added to the Table API](https://github.com/apache/iceberg/pull/4945)
+* Support for [Table scan reporting](https://github.com/apache/iceberg/pull/5268), which enables collection of statistics of the table scans.
+* Added support for Flink 1.16, dropped support for Flink 1.13
+* Dropped support for Apache Spark 3.0
+* [Fixed the SequenceNumber](https://github.com/apache/iceberg/pull/5913) to represent DataSequenceNumber in the ManifestEntry
```
Comment (rdblue) [inline landing-page/content/common/release-notes.md:79]: This wasn't a fix, it was an extension. We call the existing sequence number the "data sequence number" and added a "file sequence number" to track when files were added, even if the data is older.

Comment (ajantha-bhat) [reply landing-page/content/common/release-notes.md:79]: True, I had a file sequence number in mind but picked the wrong PR.

we can replace it with this.
Core: Add file seq number to ManifestEntry by @aokolnychyi in https://github.com/apache/iceberg/pull/6002

File: landing-page/content/common/release-notes.md (hunk 2 above)
Comment (gaborkaszab) [reply landing-page/content/common/release-notes.md:73]: Collecting and exposing scan metrics (by @nastra) was also introduced in 1.1.0 AFAIK.

Comment (Fokko) [reply landing-page/content/common/release-notes.md:73]: @ajantha-bhat thanks, that's a great suggestion. I went through the list and added them with the links to the PRs. 

@gaborkaszab Thanks for the suggestion, the PR to scan metrics is already in the list :)

File: landing-page/content/common/release-notes.md (hunk 3 above)
Comment (Fokko) [reply landing-page/content/common/release-notes.md:79]: Good catch, updated it to 6002

**Review by rdblue — APPROVED** [review]

**Comment by rdblue** [comment]
Thanks, @Fokko!

**Pull request merged**